cmd/create/operatorroles/by_prefix.go

```diff
@@ -58,7 +58,13 @@ func handleOperatorRolesPrefixOptions(r *rosa.Runtime, cmd *cobra.Command) {
 		os.Exit(1)
 	}
 	args.hostedCp = isHostedCP
-	args.installerRoleArn = interactiveRoles.GetInstallerRoleArn(r, cmd, args.installerRoleArn, "")
+	if args.hostedCp {
```

Comment:
Will this not always be true?

Reply:
The user can set the value to `false` in interactive mode.
Alternatively, if the flag isn't provided the default value will be `false`.

Reply:
I see thank you